web/src/app/registration/registration.component.scss

```diff
@@ -7,6 +7,42 @@ form, #formContainer {
     align-items: center;
     font-family: $defaultFont;
     label{
+        align-self:flex-start;
         font-weight: bold;
     }
+}
+
+input[type="text"], input[type="email"], input[type="password"]{
+    background-color: #C9FFDC;
+    width: 240px;
+    height: 30px;
+    border-radius: 6px;
+    box-shadow: 2px 2px 4px #000000;
+    border: none;
+}
+
+input[type="button"]{
```

Comment:
If the button is changed from `input type="button"` to `button` element, this can be simplified to `button { ... }`.

Reply:
Changed from `input type="button"` to `button`.